.github/workflows/ci-actions.yml

```diff
@@ -109,17 +109,17 @@ jobs:
         java :
           - { name: "8",
               java-version: 8,
-              maven_args: "-pl !integration-tests/gradle -pl !integration-tests/maven -pl !integration-tests/kubernetes/quarkus-standard-way"
+              maven_args: "-pl !integration-tests/kubernetes/quarkus-standard-way"
```

Comment:
Here I just put the common stuff in the common Maven command rather than repeating stuff for each JDK.